pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -363,6 +363,18 @@ describe('Rewards Asset manager', function () {
           { account: vault.address, changes: { DAI: ['very-near', amountToWithdraw] } },
         ]);
       });
+
+      it('allows withdrawing returns which are greater than the current managed balance', async () => {
+        await tokens.DAI.mint(assetManager.address, tokenInitialBalance.mul(10));
+        await assetManager.connect(lp).setUnrealisedAUM(amountToDeposit.add(tokenInitialBalance.mul(10)));
+
+        const amountToWithdraw = (await assetManager.maxInvestableBalance(poolId)).mul(-1);
+
+        await expectBalanceChange(() => assetManager.connect(lp).capitalOut(poolId, amountToWithdraw), tokens, [
+          { account: assetManager.address, changes: { DAI: ['very-near', -amountToWithdraw] } },
```

Comment:
```suggestion
          { account: assetManager.address, changes: { DAI: ['very-near', amountToWithdraw.mul(-1)] } },
```

Reply:
Do we need 'very-near' here? Should it be exactly equal?

Reply:
Yes, I think we can make this a strict equality. `very-near` was necessary when we were using shares but in `RewardsAssetManager` we can remove this.